src/platform/forms-system/test/js/widgets/SelectWidget.unit.spec.jsx

```diff
@@ -10,7 +10,7 @@ describe('Schemaform <SelectWidget>', () => {
     const enumOptions = [
       {
         label: 'Testing',
-        val: 'test',
+        value: 'testValue',
```

Comment:
Was this just a bug in the test?

Reply:
I think so... it should be `value` everywhere, but since all the tests were passing, I changed the actual value to `testValue` to make sure.